tests/system_tests.pm

```diff
@@ -100,6 +103,15 @@ ENDFUNC
         # upload also original xml, if something goes wrong with conversion
         upload_logs("nose2-junit.xml");
         parse_extra_log('JUnit', "nose2-junit-$test.xml");
+
+        # upload per-test logs
+        $test_logs_path = "/tmp/$test/";
+        assert_script_run("mkdir $test_logs_path");
+        assert_script_run("sudo python3 split_logs.py --junit-xml=nose2-junit-$test.xml --outdir=$test_logs_path");
+        my @files = <"$test_logs_path*">;
```

Comment:
This won't work, because the the perl script is running outside of the VM. There is a `script_output` function to run something inside the system under test and return its output.

Reply:
Ah. Makes sense. See if commit 167120b7b1c114cdd000b19a7b93750340aaf5a4 fixes it.